pegasus/migrations/137_rename_user_storage_ids.rb

```diff
@@ -0,0 +1,26 @@
+Sequel.migration do
+  up do
+    unless [:production].include?(rack_env)
```

Comment:
Mostly curious, why not just do a more straightforward equals comparison here (and in the down migration)?

Reply:
Oops this was a copy/paste that I didn't think through, there's no reason